### ISSUE TITLE
Section title padding

### ISSUE THREAD

**Issue by engmsaleh** [xmartlabs/XLForm#685]
I have added a section with header `Participant` , but it seems that it is stick to the left edge of the table .
How to get some space between the section title and its superview
![header](https://cloud.githubusercontent.com/assets/462150/12010750/f9c5daa2-acbb-11e5-8fb6-b33f0dac892b.png)

**Comment by loufq** [comment]
for now there is no inset property let you set .
then right way is : you need overload tableview's delegate menthod and do it self 
the simpliest way :you just add space perfix to fix it like  @"     PARTICIPANT"

**Comment by engmsaleh** [comment]
@loufq  I have already tried to add a white space as suggested , but it didn't work for me .
The only solution that worked from me is by overriding the TableHeaderView with the following 
```
-(UIView*)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)section
{
    if (section == 1) {
        UILabel *label = [[UILabel alloc] initWithFrame:CGRectMake(0,0,tableView.frame.size.width,30)];
        label.text = @"  Participant";
        label.backgroundColor=[UIColor clearColor];
        label.textAlignment = NSTextAlignmentLeft;
        return label;
    }
    return nil;
}
```

**Comment by mats-claassen** [comment]
Hi, by default XLForm shows the group style header of iOS.
Your solution is valid, that is the current way of creating custom headers.

**Comment by jjochen** [comment]
Setting the separator inset works without a custom header view.
```
    self.tableView.separatorInset = UIEdgeInsetsMake(0, 15, 0, 0);
    if (@available(iOS 11.0, *)) {
        self.tableView.separatorInsetReference = UITableViewSeparatorInsetFromAutomaticInsets;
    }
```
